rotkehlchen/db/history_events.py

```diff
@@ -514,7 +517,8 @@ def get_history_events_and_limit_info(
             filter_query: 'HistoryBaseEntryFilterQuery',
             has_premium: bool,
             group_by_event_ids: bool = False,
-    ) -> tuple[Union[list[tuple[int, HistoryBaseEntry]], list[HistoryBaseEntry]], int]:
+            entries_limit: Optional[int] = None,
+    ) -> tuple[Union[list[tuple[int, HistoryBaseEntry]], list[HistoryBaseEntry]], int, int]:
```

Comment:
you will need to adjust the docstring to better describe what is returned by the tuples here.